### DISCUSSION TITLE
[FEATURE REQUEST] : A "Draft" status for issues

### DISCUSSION THREAD

**Discussion by Malix-Labs** [community/community#108378] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

"Draft" issues currently only exists as projects' objects (and not issues' objects)

The best way to make them would have been to make draft issues be issues with a "Draft" status (like [Draft PRs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests)).
I'm confused as to why it was not done in the first place.

It would make things be standardized to issue objects (as they should IMO), and would automatically make
- #8328
- #10303
- #10903
natively available

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mm12** [comment]
Some orgs uses Projects' draft issues to track things that shouldn't be issues in a repository, so I feel this wouldn't solve #8328 well for those users. Though, this would also fix #46699
